provisioning/device/src/Models/RegistrationRequestPayload.cs

```diff
@@ -28,7 +28,13 @@ public class RegistrationRequestPayload
         public void SetPayload(string payloadAsJsonString)
         {
             using var jd = JsonDocument.Parse(payloadAsJsonString);
-            Payload = jd.RootElement;
+
+            // JsonDocument is IDisposable. Since the SDK needs to work with a particular element within the JSON document,
+            // we need to return the Clone of that JsonElement.
+            // If we return the RootElement or a sub-element directly without making a Clone,
```

Comment:
Same here with case of word clone:

```suggestion
            // we need to return the clone of that JsonElement.
            // If we return the RootElement or a sub-element directly without making a clone,
```